### PR TITLE
Small cleanups

### PR DESCRIPTION
**Pull request by dspicher**
Some small cleanups I noticed while working on [my fork](https://github.com/dspicher/bitcoin-blockparser).

### CODE REVIEW

File: src/blockchain/parser/index.rs [hunk 1]
```diff
@@ -130,15 +130,15 @@ impl fmt::Debug for BlockIndexRecord {
...
+    let mut block_index = HashMap::with_capacity(900000);
     let mut db_iter = DB::open(path, Options::default())?.new_iter()?;
     let (mut key, mut value) = (vec![], vec![]);
 
     while db_iter.advance() {
         db_iter.current(&mut key, &mut value);
         if is_block_index_record(&key) {
             let record = BlockIndexRecord::from(&key[1..], &value)?;
-            if record.status & (BLOCK_VALID_CHAIN | BLOCK_HAVE_DATA | BLOCK_VALID_CHAIN) > 0 {
+            if record.status & (BLOCK_VALID_CHAIN | BLOCK_HAVE_DATA) > 0 {
```
Comment (gcarq) [inline src/blockchain/parser/index.rs:141]: Is `BLOCK_VALID_CHAIN` not needed?

**Review by gcarq — COMMENTED** [review]
Thanks a lot for upstreaming the fixes

File: src/blockchain/parser/index.rs (hunk 1 above)
Comment (dspicher) [reply src/blockchain/parser/index.rs:141]: `BLOCK_VALID_CHAIN` is already the first operand to the or expression. `x || x == x`.

Comment (gcarq) [reply src/blockchain/parser/index.rs:141]: Ah yes, ofc. Missed that it was there two times

**Pull request merged**